### PR TITLE
compliance consolidation updates

### PR DESCRIPTION
**Pull request by theseyi**
- 	removes obsolete confidential feature component sources
- 	fixes issue with tracking current user immediately after login: moves user tracking to index from application. adds session invalidation when retrieving the current user fails. slightly refactors logout

### CODE REVIEW

File: wherehows-web/app/routes/index.js [hunk 1]
```diff
@@ -1,7 +1,22 @@
 import Ember from 'ember';
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+import fetch from 'ember-network/fetch';
```
Comment (justYves) [inline wherehows-web/app/routes/index.js]: fyi, looks like `ember-fetch` is replacing `ember-network` .
Based on conversations in https://github.com/samselikoff/ember-cli-mirage/issues/983, https://github.com/tomdale/ember-network/issues/17 and mostly https://github.com/stefanpenner/ember-fetch/issues/21

File: wherehows-web/app/routes/logout.js [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
 
   actions: {
+    /**
+     * Post transition, call endpoint then invalidate current session on client on success
+     */
     didTransition() {
-      Ember.$.get('/logout').then(() => {
-      this.get('session').invalidate();
-      });
+      fetch('/logout').then(() => get(this, 'session').invalidate());
```
Comment (justYves) [inline wherehows-web/app/routes/logout.js]: can this call ever fail? do we need a `catch` ?

Comment (theseyi) [inline wherehows-web/app/routes/logout.js]: If the logout fails, at this point the user will remain on the page and will probably re-attempt to logout.

File: wherehows-web/app/routes/index.js (hunk 1 above)
Comment (theseyi) [inline wherehows-web/app/routes/index.js]: 👍

**Pull request merged**